src/js/mods/modloader.js

```diff
@@ -187,16 +198,25 @@ export class ModLoader {
             this.modLoadQueue.push({
                 modClass,
                 meta,
+                path,
             });
         };
 
         mods.forEach(modCode => {
+            path = modCode.slice(0, modCode.indexOf("\n")).replace("// ", "");
```

Comment:
this is really hacky, please just store the path in a tuple with the mod code, not actually in the mod code